src/readqps.jl

```diff
@@ -29,8 +29,19 @@ mutable struct QPSData
     uvar::Vector{Float64}            # variables upper bounds
     name::String                     # problem name
     objname::String                  # objective function name
+    rhsname::String                  # Name of RHS field
+    bndname::String                  # Name of BOUNDS field
+    rngname::String                  # Name of RANGES field
     varnames::Vector{String}         # variable names, aka column names
     connames::Vector{String}         # constraint names, aka row names
+    # name => index mapping for variables
+    # Variables are indexed from 1 and onwards
+    varindices::Dict{String, Int}
```

Comment:
Isn't this redundant with `varnames`? Searching through a Dict should be more efficient than through a vector, so perhaps we don't need the `varnames` vector?!

Reply:
While building the model, we need a mapping name => index (since variables are tracked by their name in the MPS file). This is implemented in the `varindices` dictionary.

The `varnames` vector gives the reverse mapping, i.e., index => name.
Agreed, the vector `varnames` holds the same information as `varindices`, but it provides a (much) faster way to query the name of the i-th variable.